When flexdll is enabled, libasmrun.a has dependencies on its symbols
in order to implement natdynlink (runtime loading of .cmxs modules).
We don't need that here, and linking with flexlink is not an option,
so provide stubs to resolve these symbols.  (FIXME: would it be enough
to return NULL or 0?)

--- origsrc/kalzium-4.10.5/src/solver/modwrap.c	2013-06-28 12:17:40.982857100 -0500
+++ src/kalzium-4.10.5/src/solver/modwrap.c	2013-10-28 04:55:35.915490100 -0500
@@ -37,3 +37,11 @@ char* solve_equation(const char * eq)
     return strdup(String_val(callback(*solve_equation_closure, v)));
 }
 
+
+#ifdef __CYGWIN__
+#include <dlfcn.h>
+void *flexdll_dlopen (const char *dll, int flags) { return dlopen (dll, flags); }
+void *flexdll_dlsym (void *handle, const char *symbol) { return dlsym (handle, symbol); }
+int flexdll_dlclose (void *handle) { return dlclose (handle); }
+char *flexdll_dlerror (void) { return dlerror(); }
+#endif
